editor/components/document-outline/index.js

```diff
@@ -14,8 +14,8 @@ import { __ } from '@wordpress/i18n';
  */
 import './style.scss';
```

Comment:
I noticed that we don't prefix `document-outline` class name with `editor-` like in all other places. We can fix it in another PR if it isn't designed this way on purpose.

Reply:
Good catch, it should be prefixed. This is documented in coding guidelines, _but_ I'm now noticing they were written prior to other folders containing components:

https://github.com/WordPress/gutenberg/blob/master/docs/coding-guidelines.md#css

>Any default export of a folder's index.js must be prefixed with editor- followed by the directory name in which it resides:
>
>`.editor-[ directory name ]`

The intended convention is:

- `[ root module name ]-[ current directory name ]`
   - Example: `editor-foo`
- `[ root module name ]-[ current directory name ]__[ descendant description ]`
   - Example: `editor-foo__child`
- `[ root module name ]-[ current directory name ]__[ descendant description ].is-[ modifier description ]`
   - Example: `editor-foo__child.is-green`